### ISSUE TITLE
Support dependencies with package names that include ";"

### ISSUE THREAD

**Issue by li-peng-dd** [tokopedia/gripmock#25]
I noticed that if you try to start a service with a proto file that includes a dependency on something with package like `go_package=github.com/golang/protobuf/protoc-gen-go/descriptor;descriptor`, it fails with:

```
2019/09/05 03:16:17 Failed to generate server formatting 20:19: invalid import path: "github.com/golang/protobuf/protoc-gen-go/descriptor;descriptor" (and 1 more errors)
```

This is because `server.tmpl` just naively uses the package declaration as the import statement, which results in invalid like above. It can be fixed by truncating the package declaration at the semicolon.

**Comment by li-peng-dd** [comment]
@jekiapp I propose #26 as fix for this.

**Comment by jekiapp** [comment]
solved in #27
